docs/tutorial.rst

```diff
@@ -315,10 +315,16 @@ The determination between the two modes is done well ahead of any execution.
 You do not need to worry about partially executed commands - that is
 impossible.
 
-If you absolutely want to run a subprocess command, you can always
-force xonsh to do so with the syntax that we will see in the following
-sections.
+If you still have concerns about the determination process and want to have strong confidence
+when running a subprocess command in your xonsh scripts, you can always use the elegant subprocess
+mode syntax that we will see in the following sections. Short example now:
```

Comment:
The english here should be cleaned up a little

Reply:
www.grammarly.com told me that
* Grammar - Correct
* Punctuation - Correct
No thoughts what you mean :)
Please fix, you have edit rights.

Reply:
Yeah, the syntax is all correct. It is that the word choice and idioms feel a little off to me. Sorry, but I can barely keep up with all of the other issues.  I have to let other people help.

Reply:
@gforsyth Gil, please help with idioms ;)

Reply:
How about something like:

```
If you would like to explicitly run a subprocess command, you can always
use the formal xonsh subprocess syntax that we will see in the following sections. For example:
```

Reply:
done